public/editor/scripts/project-list.js

```diff
@@ -81,44 +81,88 @@ require(["jquery", "constants", "analytics", "moment"], function($, Constants, a
+      deleteBtn.className = "delete-button";
+      deleteBtn.onclick = function() {
+        // TODO: we can do better than this, but let's at least make it harder to lose data.
+        if(!window.confirm("{{ deleteProjectConfirmationText }}")) {
+          return false;
+        }
+        
+        Array.prototype.forEach.call(projectsToDelete, function(project){
+          var projectId = project.attr("data-project-id");
+          var projectElementId = project.attr("id");
+          $("#" + projectElementId + " > .project-title").off("click");
+
+          analytics.event("DeleteProject");
+
+          var request = $.ajax({
+            headers: {
+              "X-Csrf-Token": $("meta[name='csrf-token']").attr("content")
+            },
+            type: "DELETE",
+            url: "/" + locale + "/projects/" + projectId,
+            timeout: Constants.AJAX_DEFAULT_TIMEOUT_MS
+          });
+          request.done(function() {
+            if(request.status !== 204) {
+              console.error("[Thimble error] sending delete request for project ", projectId, request.status);
+            }
+          });
+          request.fail(function(jqXHR, status, err) {
+            err = err || new Error("unknown network error");
+            console.error(err);
+          });
+
+          project.hide({
+            duration: 250,
+            easing: "linear",
+            done: function() {
+              project.remove();
+            }
+          });
+        });
+        projectsToDelete = [];
+        $(".delete-button").remove();
+      };
+      var listWrapper = document.getElementsByClassName("project-list-wrapper");
+      listWrapper[0].appendChild(deleteBtn);
     }
 
     var project = $(this).closest(".project");
-    var projectId = project.attr("data-project-id");
-    var projectElementId = project.attr("id");
-    $("#" + projectElementId + " > .project-title").off("click");
-
-    analytics.event({ category : analytics.eventCategories.PROJECT_ACTIONS, action : "Delete Project" });
-
-    var request = $.ajax({
-      headers: {
-        "X-Csrf-Token": $("meta[name='csrf-token']").attr("content")
-      },
-      type: "DELETE",
-      url: "/" + locale + "/projects/" + projectId,
-      timeout: Constants.AJAX_DEFAULT_TIMEOUT_MS
-    });
-    request.done(function() {
-      if(request.status !== 204) {
-        console.error("[Thimble error] sending delete request for project ", projectId, request.status);
+    var projectSelector = "#" + project.attr("id");
+    var deleteIndex = projectsToDelete.findIndex(function(x) {return x.attr("id") === project.attr("id");});
+    var disabledLink = {"cursor":"default", "text-decoration":"none"};
+    var enabledLink = {"cursor":"pointer", "text-decoration":"underline"};
+    
+    if(deleteIndex === -1) {
+      projectsToDelete.push(project);
+      $(projectSelector + " .edit-link").bind("click", false).css({"cursor":"default"});
```

Comment:
I was not able to use `.off("click")` since the links are all anchor tags and I don't define a `click` behavior so using `.off("click")` did not seem to effect them at all. I found that the common approach to disabling the href links was to set a `click` behavior to use `.preventDefault()`. This `click` behavior is then turned off to activate the link again as can be seen in my latest changes.